fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/data/LaoHandler.java

```diff
@@ -125,6 +138,7 @@ public static void handleUpdateLao(HandlerContext context, UpdateLao updateLao)
    * @param context the HandlerContext of the message
    * @param stateLao the message that was received
    */
+  @SuppressLint("CheckResult")
```

Comment:
why do we need this?

Reply:
It's because we do not use the result of the subscribe (the javaRx one) which returns a disposable. For lifecycle aware parts of the codebase, those disposables are stored and then disposed of at the end of the lifecycle. 
In all our handlers those disposables are ignored. The suppressed lint is to make the IDE less aggressive about it.

I think we should provide the disposable container of the `LaoNetworkManager` to the handler so that they can add their disposables to it. What do you think @GabrielFleischer ?

I think it should be done in a separate PR. I can do it fairly quickly :)